apidoc/Titanium/Locale/Locale.yml

```diff
@@ -235,3 +235,14 @@ properties:
         sections of wikipedia for reference.
     type: String
     permission: read-only
+
+  - name: applicationLocales
+    summary: Update or retrieve Android's Per-app languages.
+    platforms: [android]
+    description: |
+        This property holds the values of Android's Per-app languages which is an array of defined Per-app languages, 
+        if no languages defined, an empty array will be returned. You can set the value of it like this `en-US,en-GB,ar-SA`.
+        This property requires Android API level 33 and above.
+        See the [Android's Per-app languages](https://developer.android.com/guide/topics/resources/app-languages).
+    type: Array
```

Comment:
Add the array typings

Reply:
can you please explain

Reply:
See the newer comments to split the API. But in general, you just have an array as the return value, but really it's `Array<LocaleDetails>` to document all properties. This should not just refer to the Android docs, as there may be newer properties in the future that are not exposed in Titanium, yet.